src/HttpTransport.cpp

```diff
@@ -10,21 +9,22 @@ using namespace apache::thrift;
 
 namespace cppkin
 {
+
+#ifdef THRIFT_FOUND
```

Comment:
We need to think on a better way to make Thrift optional, like it was done in Core with SPDLOG, i don't like the **#ifdef** style inside a code.